src/amo/components/ErrorPage/NotFound/index.js

```diff
@@ -26,33 +26,33 @@ export class NotFoundBase extends React.Component<InternalProps> {
   render() {
     const { i18n } = this.props;
 
-    // We use `getLocalizedTextWithLinkParts()` two times to create all the
-    // variables needed to display both the `Link` components and texts
-    // before/after.
-    let linkParts = getLocalizedTextWithLinkParts({
-      i18n,
+    const paragraphWithLinks = replaceStringsWithJSX({
       text: i18n.gettext(
         `Try visiting the page later, as the theme or extension may become
         available again. Alternatively, you may be able to find what you’re
         looking for in one of the available %(linkStart)sextensions%(linkEnd)s
         or %(secondLinkStart)sthemes%(secondLinkEnd)s.`,
       ),
-      // By setting `linkStart`/`linkEnd` here, we can reuse
-      // `getLocalizedTextWithLinkParts()` directly after.
-      otherVars: {
-        secondLinkStart: '%(linkStart)s',
-        secondLinkEnd: '%(linkEnd)s',
+      replacements: {
```

Comment:
Yep, that's much better. I wasn't happy with the `getReplacementKey()` trick. I had to use that because `{ [['foo', 'bar']]: () => {} }` isn't supported by Flow -__-